textproto/header.go

```diff
@@ -488,6 +489,12 @@ const (
 	maxLineOctets  = 4000
 )
 
+var (
+	ErrMalformedMimeHeaderInittialLine = errors.New("message: malformed MIME header initial line")
```

Comment:
Is there a reason why this error should be a separate value and therefore part of API? (As opposed to being the same as the second one)